src/main/java/com/bytecoders/pharmaid/PharmaidController.java

```diff
@@ -38,6 +42,9 @@ public class PharmaidController {
   @Autowired
   private PrescriptionService prescriptionService;
 
+  private static final String USER_ID = "userId";
+  private static final String PRESCRIPTION_ID = "prescriptionID";
```

Comment:
typo with "prescriptionID", last "D" should be uppercase.  Received `Exception: Required URI template variable 'prescriptionID' for method parameter type String is not present]` error when pulling down the branch